apis/betterself/v1/supplements/serializers.py

```diff
@@ -109,11 +122,19 @@ def validate(self, validated_data):
 
         if 'ingredient_compositions' in validated_data:
             ingredient_compositions_uuids = validated_data.pop('ingredient_compositions')
-            ingredient_compositions = IngredientComposition.objects.filter(uuid__in=ingredient_compositions_uuids)
 
-            if ingredient_compositions.count() != len(ingredient_compositions_uuids):
-                raise ValidationError('Not all ingredient composition UUIDs were found {}'.format(
-                    ingredient_compositions_uuids))
+            if isinstance(ingredient_compositions_uuids, list):
```

Comment:
Kinda hate this hack now, but it makes eventually life easier for other developers ...